pkg/operator/operator.go

```diff
@@ -98,6 +99,7 @@ type Operator struct {
 	logger  logr.Logger
 	opts    Options
 	client  client.Client
+	getCA   func() (*x509.Certificate, error)
```

Comment:
nit: is this used anywhere? I couldn't find it.

Reply:
Now it is. 😄